README.md

```diff
@@ -0,0 +1,21 @@
+# permissive-script-security-plugin
+Permissive Script Security Plugin
+
+Turn on permissive mode of Script Security Plugin. Problematic
```

Comment:
```suggestion
Turns on permissive mode of Script Security Plugin. Problematic
```

Reply:
Would it be better to use "Enables" rather than using the phrasal verb "Turns on"?
```suggestion
Turn on permissive mode of Script Security Plugin. Problematic
```